kaldb/src/main/java/com/slack/kaldb/logstore/search/SearchResultAggregatorImpl.java

```diff
@@ -39,7 +39,7 @@ public SearchResultAggregatorImpl(SearchQuery searchQuery) {
     int totalNodes = 0;
     int totalSnapshots = 0;
     int snapshpotReplicas = 0;
-    int totalCount = 0;
+    long totalCount = 0;
```

Comment:
What's the reason behind changing these? I'm not sure why we're moving some of the counters to doubles or long values, when these will never contain decimal values, or exceed the int overflow. It seems like perhaps the reason these are being flagged is due to issues elsewhere?

Reply:
The tool flagged them as high risk, so changed them. These are legitimate changes though since now there are no down conversions and all conversions are explicit.